bin/spring

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+# This file loads spring without using Bundler, in order to be fast.
+# It gets overwritten when you run the `spring binstub` command.
+
+unless defined?(Spring)
+  require "rubygems"
```

Comment:
__[staging][rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.